src/players/services/players.service.spec.ts

```diff
@@ -286,6 +286,25 @@ describe('PlayersService', () => {
       });
     });
 
+    describe('when a super-user tries logging in', () => {
+      beforeEach(() => {
+        environment.superUser = 'FAKE_STEAM_ID_2';
+      });
+
+      it('should force create the account', async () => {
+        const forceCreateSpy = jest.spyOn(service, 'forceCreatePlayer');
+
+        await service.createPlayer(mockSteamProfile);
+
+        expect(forceCreateSpy).toHaveBeenCalled();
```

Comment:
Don't test whether a particular method was called, test the result.

Reply:
Mock `Etf2lService.fetchPlayerInfo()` so that it throws an error; do so for `SteamApiService.getTf2InGameHours()` as well.
Also, test whether the `roles` property are correct.